test/models/v3/message.spec.ts

```diff
@@ -26,20 +20,6 @@ describe('Message model', function() {
     });
   });
 
-  describe('.schemaFormat()', function() {
```

Comment:
Why remove the tests when we still have the functions? 🤔

Reply:
The reality is that message **should not** have the `schemaFormat` method but only the MessageTrait. See https://github.com/asyncapi/parser-api/pull/86/files#diff-ac35ac6e0b0ad3629802a0e04076aa18e94da614f5392e1164dcaacdfe3ee198L157